src/js/models/v2/awardsV2/BaseContract.js

```diff
@@ -3,6 +3,7 @@
  * Created by David Trinh 10/9/18
  */
 import * as MoneyFormatter from 'helpers/moneyFormatter';
+import * as pscHelper from 'helpers/pscHelper';
```

Comment:
I prefer `import { emptyHierarchy, deducePscType } from 'helpers/pscHelper';`, as it is clearer.